spec/kitchen_spec/directions/bake_iframes/v1_spec.rb

```diff
@@ -3,76 +3,64 @@
 require 'spec_helper'
 
 RSpec.describe Kitchen::Directions::BakeIframes::V1 do
-  let(:book_with_div_media) do
+  let(:book_with_iframes_in_note_and_list) do
     book_containing(html:
-      one_chapter_with_one_page_containing(
-        <<~HTML
-          <div data-alt="atoms_isotopes" data-type="media" id="auto_78a58d46-6a29-4972-b848-385be0c8057f_11">
-            <iframe height="371.4" src="url" width="660">
-          <!-- no-selfclose -->
-            </iframe>
+      <<~HTML
+        <span data-type="slug" data-value="the-book-slug">
+        <div data-type="chapter">
+          <div data-type="page" class="introduction"></div>
+          <div data-type="page" id="page_1234">
+            <h1 data-type="document-title">The Document: Title!</h1>
+            <div class="interactive" data-has-label="true" data-label="" data-type="note" id="iframenote">
+              <div data-alt="atoms_isotopes" data-type="media" id="media1">
+                <iframe height="371.4" src="https://openstax.org/l/atoms_isotopes" width="660"><!-- no-selfclose -->
+                  </iframe>
+              </div>
+            </div>
+            <div class="interactive interactive-long" data-has-label="true" data-label="" data-type="note" id="iframenote3">
+              <ul>
+                <li>1: The evolution from fish to earliest tetrapod<span data-type="newline"><br /></span>
+                  <div data-alt="tetrapod_evol1" data-type="media" id="media2"><iframe height="371.4" src="url1" width="660"><!-- no-selfclose --></iframe></div>
+                </li>
+                <li>2: The discovery of coelacanth and <em data-effect="italics">Acanthostega</em> fossils<span data-type="newline"><br /></span>
+                  <div data-alt="tetrapod_evol2" data-type="media" id="media3"><iframe height="371.4" src="url2" width="660"><!-- no-selfclose --></iframe></div>
+                </li>
+              </ul>
+            </div>
           </div>
-        HTML
-      )
+        </div>
+      HTML
     )
   end
 
-  let(:book_with_internal_resource) do
+  let(:book_with_baked_iframes) do
     book_containing(html:
-      one_chapter_with_one_page_containing(
-        <<~HTML
-          <div data-alt="atoms_isotopes" data-type="media" id="auto_78a58d46-6a29-4972-b848-385be0c8057f_11">
-            <iframe height="371.4" src="../resources/Interactive/090221_opx_comp_ch04_1a_drag_and_drop_labeling/index.html" width="660">
+      <<~HTML
+        <div data-type="chapter"><div data-type="page" id="page_4567">
+          <div class="os-has-iframe os-has-link" data-type="alternatives"><a class="os-is-link" href="abc">Click to view content</a>
+            <iframe height="371.4" src="abc" width="660" class="os-is-iframe">
           <!-- no-selfclose -->
             </iframe>
           </div>
-        HTML
-      )
+        </div></div>
+      HTML
     )
   end
 
-  let(:book_with_note) do
+  let(:book_with_error_prone_iframe) do
```

Comment:
I'm not fully understand why in this book case the iframe is proning error :sweat_smile: